extensions/purchase-label/firebase.json

```diff
@@ -1,17 +0,0 @@
-{
-    "functions": {
-        "source": "functions",
-        "predeploy": "npm --prefix \"$RESOURCE_DIR\" run build"
-    },
-    "emulators": {
-        "functions": {
-            "port": 5001
-        },
-        "firestore": {
-            "port": 8080
-        },
-        "ui": {
-            "enabled": true
-        }
-    }
-}
```

Comment:
Why are we removing the firebase.json? Are you sure the emulator still works without this? I was unable to get it to work without this previously.

Reply:
They're not needed and I found some examples of location `.env` and firebase config files near shared test setup files.